<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VCRS485_Config</class>
 <widget class="QDialog" name="VCRS485_Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конфигурация датчика напряжения</string>
  </property>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>335</y>
     <width>95</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>455</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string>Датчик AnK №1</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Значение коэффициента K</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="koeff_K">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>120</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-1000000000</number>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="koeff_Z">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>55</y>
      <width>120</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-1000000000</number>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Значение коэффициента Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Размерность параметра</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dim">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>120</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>122</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Десятичный знак</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dec">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>122</y>
      <width>40</width>
      <height>26</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="dWrite">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>95</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Записать</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>165</y>
     <width>455</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string>Датчик AnK №2</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Значение коэффициента K</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="koeff_K_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>120</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-1000000000</number>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="koeff_Z_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>55</y>
      <width>120</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-1000000000</number>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Значение коэффициента Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Размерность параметра</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dim_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>120</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>122</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Десятичный знак</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dec_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>122</y>
      <width>40</width>
      <height>26</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="dWrite_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>95</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Записать</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
